src/authorization/authorization.ts

```diff
@@ -273,7 +273,7 @@ export function initialize(
             */
             if (
               !!(config?.url || '').match(
-                /(users\/update)|(events\/trackInApp)|(events\/inAppConsume)|(events\/track)/gim
+                /(users\/update)|(events\/trackInApp)|(events\/inAppConsume)|(events\/track)|(events\/received)|(events\/click)/gim
```

Comment:
should we do this for tracking impressions too?

Reply:
Done. Will resolve conflicts now